File: Yogscart Alpha/Assets/Scripts/KartMaker Scripts/CharacterSkeleton.js
#pragma strict

//Character Skeleton
//By Robo_Chiz V1

//Locations of the axles for each wheel.
var SeatPosition : Vector3; //Represented by a Cyan Box
var HatHolder : Transform; //Represented by a Magenta Box

	function OnDrawGizmos() {
		
		Gizmos.color = Color.cyan;
		Gizmos.DrawSphere(transform.position + SeatPosition,0.1);
		
		Gizmos.color = Color.magenta;
		Gizmos.DrawCube(HatHolder.position,Vector3(0.5,0.05,0.5));
			
	}

File: Yogscart C#/Assets/Scripts/Kart Maker/Character_Skeleton.cs
using UnityEngine;
using System.Collections;

//Character Skeleton
//By Robo_Chiz V2

public class Character_Skeleton : MonoBehaviour {

    //Locations of the axles for each wheel.
    public Vector3 SeatPosition; //Represented by a Cyan Box
    public Transform HatHolder; //Represented by a Magenta Box

	void OnDrawGizmos()
    {

        Gizmos.color = Color.cyan;
        Gizmos.DrawSphere(transform.position + SeatPosition, 0.1f);

        Gizmos.color = Color.magenta;
        Gizmos.DrawCube(HatHolder.position, new Vector3(0.5f, 0.05f, 0.5f));

    }
}
